### ISSUE TITLE
Accessibility ♿: Missing Tooltip and Title for Toggle Dark Mode Icon

### ISSUE THREAD

**Issue by tobidosumu** [mozilla/perfcompare#411]
### Description

The toggle dark mode icon on the PerfCompare web app is missing both a tooltip and an alt text, which can negatively impact accessibility for users with visual impairments or who rely on assistive technologies such as screen readers. To improve its accessibility, we need to add appropriate tooltip and title text for the toggle dark mode icon.

### Expected Outcome

- Add a tooltip to the toggle dark mode icon to describe its function.

- Add an `title` attribute to the toggle dark mode icon with an appropriate description, such as "Toggle dark mode".

- Test PerfCompare using assistive technologies to ensure that the tooltip and alt text are functioning properly.
